python/cudf/cudf/_lib/pylibcudf/libcudf/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@
 set(cython_sources aggregation.pyx binaryop.pyx copying.pyx replace.pyx reduce.pxd
                    stream_compaction.pyx types.pyx unary.pyx
 )
+add_subdirectory(strings)
```

Comment:
For consistency with other files I'd suggest putting this after the `rapids_cython_create_modules` call, but it doesn't really matter.